tests/libtest/thrift_client_to_db.py

```diff
@@ -81,30 +79,18 @@ def __exit__(self, type, value, tb):
         self._transport.close()
 
 
-class CCReportHelper(ThriftAPIHelper):
-
-    def __init__(self, host, port, auto_handle_connection=True):
-        # import only if necessary; some tests may not add this to PYTHONPATH
-        from DBThriftAPI import CheckerReport
-
-        transport = TTransport.TBufferedTransport(TSocket.TSocket(host, port))
-        protocol = TBinaryProtocol.TBinaryProtocol(transport)
-        client = CheckerReport.Client(protocol)
-        super(CCReportHelper, self).__init__(transport, client,
-                                             auto_handle_connection)
-
-
 class CCViewerHelper(ThriftAPIHelper):
 
-    def __init__(self, host, port, uri, auto_handle_connection=True,
-                 session_token=None):
-        # import only if necessary; some tests may not add this to PYTHONPATH
+    def __init__(self, host, port, product, endpoint,
+                 auto_handle_connection=True, session_token=None):
+        # Import only if necessary; some tests may not add this to PYTHONPATH.
```

Comment:
Importing a module multiple times is not a problem [docs](https://docs.python.org/2/tutorial/modules.html#more-on-modules). It only gives a [performance overhead](https://stackoverflow.com/a/29504318/1428773) on name resolution.

> A module can contain executable statements as well as function definitions. [...] They are executed only the first time the module name is encountered in an import statement.

But even if importing the same module multiple times would execute, the things imported here are functions and constant, which are stateless.